MFRC522.h

```diff
@@ -78,6 +78,8 @@
 #include <Arduino.h>
 #include <SPI.h>
 
+#define MFRC522_SPICLOCK (F_CPU / 2)			// Fastest possible speed (MFRC522 accept upto 10MHz)
```

Comment:
I think that use 2 bytes of RAM to save SPI speed is somehow a waste because is rarely need to change while running. How about use a preprocessor to choose between static SPI speed and changable one?